### ISSUE TITLE
Data List a11y violations

### ISSUE THREAD

**Issue by hxltrhuxze** [Royal-Navy/design-system#1097]
- [ ] Ensures `<dl>` elements are structured correctly
- [ ] Ensures `<dt>` and `<dd>` elements are contained by a `<dl>`

**Comment by m7kvqbe1** [comment]
Current implementation is actually valid - see MDN here. I've closed this ticket for now.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

<img width="1064" alt="Screenshot 2020-08-18 at 09 23 01" src="https://user-images.githubusercontent.com/48086589/90488907-7f638900-e134-11ea-8fba-33ee85a2d0e7.png">
